packages/api/db/migration/20230124222902_update_support_type_enum.js

```diff
@@ -0,0 +1,15 @@
+export const up = async function (knex) {
+  await knex.schema.alterTable('supportTicket', () => {
+    knex.raw(`ALTER TYPE support_type ADD VALUE 'Delete my farm'`);
```

Comment:
According to the knex documentation (https://knexjs.org/guide/schema-builder.html#enum-enu) "Knex does not provide any way to alter enumerations after creation. To change an enumeration later on you must use Knex.raw, and the appropriate command for your database.". I looked up how to do so and found this syntax. 

NOTE: this did not work. It did not throw an error when the migration ran, but when I went and looked at the constraint check, it had not been updated. I ran this directly against the database and got an error that support type does not exist. 

Truthfully, I have never used Postgres (I've only used MySQL) and have always been told by those more experienced than me to stay away from the enum type when using MySQL.